docs/Installation/Installation_Guide.rst

```diff
@@ -102,315 +96,29 @@ If you are behind a proxy, add the following lines to your local machine’s
     https_proxy=http://user;pass@<my-ip>:<my-port>
     no_proxy=localhost,127.0.0.1,10.*
 
-******************************
```

Comment:
Is all this deleted stuff getting moved the metalk8s repo?

Reply:
That is my hope.